src/napari_imagej/java.py

```diff
@@ -215,6 +215,11 @@ def _validate_imagej(self):
             (jc.Module, "org.scijava:scijava-common", "2.91.0"),
             (jc.Searcher, "org.scijava:scijava-search", "2.0.0"),
         ]
+        # Add additional minimum versions for legacy components
+        if ij().legacy and ij().legacy.isActive():
+            component_requirements.append(
+                (ij().legacy.getClass(), "net.imagej:imagej-legacy", "1.0.1")
```

Comment:
@gselzer This needs to be 1.1.0, not 1.0.1. Because your change imagej/imagej-legacy@cc161da66a38609ca4b3461397b99adafd187e66 added new public API, so by SemVer, the minor digit must be advanced.